README.md

```diff
@@ -72,8 +72,7 @@ This will create a new Rust project directory with the following structure:
 As an example, you can change the content of `./src/main.rs` to:
 
 ```rust
-#![no_std]
-#![no_main]
+#![cfg_attr(target_arch = "riscv32", no_std, no_main)]
```

Comment:
Just noting for the future that this is an additional motivating to add our own custom target to differentiate vanilla `riscv32` from our NVM-variant that will include, e.g., precompiles

Reply:
I don't follow the motivation, having precompiles doesn't necessarily change the architecture of the target. Besides, CTRL+SHIFT+R "riscv32" to "my_nexus_target".

Reply:
Oh this was a note for the future, not something that should in anyway change this PR.

Was just noting cause it occurred to me that as the VM starts to differ in behavior from `riscv32` more generally, that we should probably make sure the UX reflects that for explicitness.